src/braket/circuits/result_types.py

```diff
@@ -181,7 +181,8 @@ def __init__(
         target: list[QubitSetInput] | None = None,
         parameters: list[Union[str, FreeParameter]] | None = None,
     ):
-        """
+        """Inits an `AdjointGradient`.
```

Comment:
All these "Inits ..." are unhelpful and are getting annoying . Can we configure ruff to not require a description for init methods?

Reply:
We can work on noqas for that but the guidance is that class constructors should have docstrings https://peps.python.org/pep-0257/. In some of these cases, the docstrings should be more verbose but have not been typed out.

Reply:
Bad/unhelpful documentation is worse than no documentation.